packages/react-native/ReactCommon/React-FabricComponents.podspec

```diff
@@ -137,7 +137,8 @@ Pod::Spec.new do |s|
       sss.dependency             folly_dep_name, folly_version
       sss.compiler_flags       = folly_compiler_flags
       sss.source_files         = "react/renderer/components/scrollview/**/*.{m,mm,cpp,h}"
-      sss.exclude_files        = "react/renderer/components/scrollview/tests"
+      sss.exclude_files        = "react/renderer/components/scrollview/tests",
+                                 "react/renderer/components/scrollview/platform/android"
```

Comment:
Any way we can invert this to be allowlist for iOS and base instead?

Reply:
> Any way we can invert this to be allowlist for iOS and base instead?

Hi , can you explain more? should we share the Android horizontal scroll content view for iOS?

Reply:
I mean, instead of excluding `tests`, and `android` platform, we only include shared (not in nested in scrollview folder), and iOS specific files (those in platform/ios).

Some React Native forks end up adding additional platforms, so the pattern of excluding specific ones can work in RN repo, but cause issues for those forks (which then end up including code for multiple platforms).

Reply:
@NickGerleman Got it :), I updates the podspec to only include shared files for iOS, because now we don't have iOS specific files. Please review it again.